[//]: # (Title/laboratory name)
[//]: # (How to edit: Write your title after "#")
# Kjems's lab - Aarhus

[//]: # (Laboratory text)
[//]: # (How to edit: Write a short description below this)
The Kjems`s lab is located at the Interdisciplinary Nanoscience Center (INANO) and at the Department of Molecular Biology and Genetics (MBG), both parts of Aarhus University. The research in the lab mainly focus on constructing functionalized DNA and RNA nanostructures, understanding the biogenesis and function of non-coding RNAs and the creation of delivery systems for gene medicine. The exosome study was started from 2015 to investigate the RNA composition and function of exosomes in tissue regeneration. There are two postdocs, Yan Yan and Ping Song,  involved in the exosome projects.

[//]: # (Layout - Linebreak \(Do not remove\)) 
<br /> 




[//]: # (Subtitle/technical platform - Try to keep consistent with other groups/descriptions.)
## Technical platform

[//]: # (Apparatus name)
[//]: # (How to edit: Write name of apparatus after "###" and edit image by altering the path after "img scr=". Do not edit align or height.)
### EV Characterization <img src="https://github.com/AndersAskeland/DSEV/blob/main/graphics/placeholder.png" align="right" height="150"/>

[//]: # (Apparatus description)
[//]: # (How to edit: Write description of method below this) 
We have a range platforms for EV of characterization including Dynamic light scattering (DLS), Nanoparticle tracking analysis (NTA), Atomic force microscopy (AFM), Electron Microscopy (EM), Flow cytometry.  

<br /> 

### EV-omics <img src="https://github.com/AndersAskeland/DSEV/blob/main/graphics/placeholder.png" align="right" height="150"/>
* We have rich experience in RNA purification and RNA sequencing library preparation (small RNA seq or long RNA seq) for exosome samples. Relevant equipment include bioanalyzer, pippin prep machine, Illumina and Oxford Nanopore sequencing technologies, and bioinformatic pipelines running in our lab.
* Omiics, spin out company from Kjems`s lab, developed pipeline for tRNA data analysis (sequencing data).
* Using DNA and RNA aptamers for pulling down and profling of exsosomes.


[//]: # (Layout - Linebreak \(Do not remove\)) 
<br />




[//]: # (Subtitle/research staff - Try to keep consistent with other groups/descriptions.)
## Research staff

[//]: # (Person name)
[//]: # (How to edit: Write person name after "###". Write location of image after "img scr=". Do not edit align, height or width.)
### Jørgen Kjems <img src="https://github.com/AndersAskeland/DSEV/blob/main/graphics/placeholder_staff.png" align="right" height="150" width="120"/>

[//]: # (Person title)
[//]: # (How to edit: Write title of person after "####".) 
#### Professor

[//]: # (Person description)
[//]: # (How to edit: Write description of person below this) 
Email: <jk@mbg.au.dk>

Webpage: <https://inano.au.dk/about/research-groups/nanomedicine-joergen-kjems-group/>


### Yan Yan <img src="https://github.com/AndersAskeland/DSEV/blob/main/graphics/placeholder_staff.png" align="right" height="150" width="120"/>
#### Postdoc
Email: <yanyan@inano.au.dk>

Webpage: <https://www.linkedin.com/in/yan-yan-1b3b85142/> & <https://pure.au.dk/portal/en/persons/yan-yan(b9db242b-cc45-465c-a2fd-9dee4338232b).html>

### Ping Song <img src="https://github.com/AndersAskeland/DSEV/blob/main/graphics/placeholder_staff.png" align="right" height="150" width="120"/>
#### Postdoc
Email: <psong@inano.au.dk>

Webpage: <https://pure.au.dk/portal/en/persons/ping-song(7562012b-6598-4dfe-8196-6619609b818b)/publications.html>

### Morten T. Venø <img src="https://github.com/AndersAskeland/DSEV/blob/main/graphics/placeholder_staff.png" align="right" height="150" width="120"/>
#### CEO of omiics
Email: <morten.veno@omiics.com>

Webpage: <https://www.linkedin.com/in/morten-ven%C3%B8/>



[//]: # (Layout - Linebreak \(Do not remove\)) 
<br />





[//]: # (Subtitle/Publications - Try to keep consistent with other groups/descriptions.)
[//]: # (How to edit: Write article name within [] and link within \(\). Use APA biblography/reference style.) 
## Select publications
[Just, J., Yan, Y., Farup, J., Sieljacks, P., Sloth, M., Venø, M., Gu, T., de Paoli, F. V., Nyengaard, J. R., Bæk, R., Jørgensen, M. M., Kjems, J., Vissing, K., & Drasbek, K. R. (2020). Blood flow-restricted resistance exercise alters the surface profile, miRNA cargo and functional impact of circulating extracellular vesicles. Scientific reports, 10(1), 5835. https://doi.org/10.1038/s41598-020-62456-3](https://pubmed.ncbi.nlm.nih.gov/32245988/)

[Yan, Y., Chang, C. C., Venø, M. T., Mothershead, C. R., Su, J., Kjems, J. Isolating, Sequencing and Analyzing Extracellular MicroRNAs from Human Mesenchymal Stem Cells. <em>J. Vis. Exp.</em> (145), e58655, doi:10.3791/58655 (2019).](https://www.jove.com/t/58655/isolating-sequencing-analyzing-extracellular-micrornas-from-human)

[Greisen, S. R., Yan, Y., Hansen, A. S., Venø, M. T., Nyengaard, J. R., Moestrup, S. K., Hvid, M., Freeman, G. J., Kjems, J., & Deleuran, B. (2017). Extracellular Vesicles Transfer the Receptor Programmed Death-1 in Rheumatoid Arthritis. Frontiers in immunology, 8, 851. https://doi.org/10.3389/fimmu.2017.00851](https://pubmed.ncbi.nlm.nih.gov/28791012/)

[Yan, Y., Chang, C., Su, J., Venø, M. T., & Kjems, J. (2020). Osteoblastogenesis Alters Small RNA Profiles in EVs Derived from Bone Marrow Stem Cells (BMSCs) and Adipose Stem Cells (ASCs). Biomedicines, 8(10), 387. https://doi.org/10.3390/biomedicines8100387](https://pubmed.ncbi.nlm.nih.gov/32998458/)


[//]: # (Layout - Linebreak \(Do not remove\)) 
<br />





[//]: # (Subtitle/About - Try to keep consistent with other groups/descriptions.)
[//]: # (How to edit: Write appropriate info after the "*".) 
## About
* Location: Aarhus
* Webpage: [Kjems's griyo](https://inano.au.dk/about/research-groups/nanomedicine-joergen-kjems-group/)
